src/plonk/mod.rs

```diff
@@ -154,6 +154,11 @@ pub struct PlonkStructure<F: PrimeField> {
 
     pub(crate) custom_gates_lookup_compressed: CompressedGates<F>,
 
+    /// we use uncompressed gates instead of
+    /// custom_gates_lookup_compressed in protogalaxy folding scheme
+    #[serde(skip_serializing)]
```

Comment:
Why aren't gates serialized here? If this is a temporary solution, we should flag this with TODO #20